cryptoprotocol/src/main/java/org/cryptimeleon/incentivesystem/cryptoprotocol/model/IncentivePublicParameters.java

```diff
@@ -0,0 +1,50 @@
+package org.cryptimeleon.incentivesystem.cryptoprotocol.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import lombok.experimental.NonFinal;
+import org.cryptimeleon.craco.common.PublicParameters;
+import org.cryptimeleon.craco.prf.aes.AesPseudorandomFunction;
+import org.cryptimeleon.craco.sig.sps.eq.SPSEQSignatureScheme;
+import org.cryptimeleon.math.serialization.Representation;
+import org.cryptimeleon.math.serialization.annotations.ReprUtil;
+import org.cryptimeleon.math.serialization.annotations.Represented;
+import org.cryptimeleon.math.structures.groups.GroupElement;
+import org.cryptimeleon.math.structures.groups.elliptic.BilinearGroup;
+
+/**
+ * a class representing the public parameters of the 2020 incentive system
+ */
+@Value
+@AllArgsConstructor
+public class IncentivePublicParameters implements PublicParameters {
+    @NonFinal
```

Comment:
That's exactly why I did. I still think it simplifies data classes a lot, especially since we do not have to maintain boilerplate code (e.g. equals if new fields are added to a class)